From dbe9244c95b1560222c7c1c349f9619965ac1384 Mon Sep 17 00:00:00 2001
From: Ivan Vecera <ivecera@redhat.com>
Date: Wed, 19 Nov 2014 16:13:02 -0500
Subject: [ethernet] be2net: remove multiple blank lines

Message-id: <1416413587-12091-15-git-send-email-ivecera@redhat.com>
Patchwork-id: 100332
O-Subject: [RHEL7.1 PATCH 14/19] be2net: remove multiple blank lines
Bugzilla: 1165755
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

BZ: #1165755

Upstream commit(s):
commit d6f5473c4250f972eab2feb3faa1c923ec245124
Author: Kalesh AP <kalesh.purayil@emulex.com>
Date:   Fri Sep 19 15:46:57 2014 +0530

    be2net: remove multiple blank lines

    This patch removes multiple blank lines in the driver as per checkpatch
    warnings.

    Signed-off-by: Kalesh AP <kalesh.purayil@emulex.com>
    Signed-off-by: Sathya Perla <sathya.perla@emulex.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Ivan Vecera <ivecera@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/emulex/benet/be_cmds.c b/drivers/net/ethernet/emulex/benet/be_cmds.c
index 8dd4975..bc47d17 100644
--- a/drivers/net/ethernet/emulex/benet/be_cmds.c
+++ b/drivers/net/ethernet/emulex/benet/be_cmds.c
@@ -676,7 +676,6 @@ int be_fw_wait_ready(struct be_adapter *adapter)
  return -1;
 }
 
-
 static inline struct be_sge *nonembedded_sgl(struct be_mcc_wrb *wrb)
 {
  return &wrb->payload.sgl[0];
diff --git a/drivers/net/ethernet/emulex/benet/be_ethtool.c b/drivers/net/ethernet/emulex/benet/be_ethtool.c
index 42e1a7d..feb0375 100644
--- a/drivers/net/ethernet/emulex/benet/be_ethtool.c
+++ b/drivers/net/ethernet/emulex/benet/be_ethtool.c
@@ -333,7 +333,6 @@ static int be_get_coalesce(struct net_device *netdev,
  struct be_adapter *adapter = netdev_priv(netdev);
  struct be_aic_obj *aic = &adapter->aic_obj[0];
 
-
  et->rx_coalesce_usecs = aic->prev_eqd;
  et->rx_coalesce_usecs_high = aic->max_eqd;
  et->rx_coalesce_usecs_low = aic->min_eqd;
diff --git a/drivers/net/ethernet/emulex/benet/be_main.c b/drivers/net/ethernet/emulex/benet/be_main.c
index 25f7fb6..f8fc455 100644
--- a/drivers/net/ethernet/emulex/benet/be_main.c
+++ b/drivers/net/ethernet/emulex/benet/be_main.c
@@ -122,7 +122,6 @@ static const char * const ue_status_hi_desc[] = {
  "Unknown"
 };
 
-
 static void be_queue_free(struct be_adapter *adapter, struct be_queue_info *q)
 {
  struct be_dma_mem *mem = &q->dma_mem;
@@ -1486,7 +1485,6 @@ static void be_eqd_update(struct be_adapter *adapter)
    tx_pkts = txo->stats.tx_reqs;
   } while (u64_stats_fetch_retry_bh(&txo->stats.sync, start));
 
-
   /* Skip, if wrapped around or first calculation */
   now = jiffies;
   if (!aic->jiffies || time_before(now, aic->jiffies) ||
-- 
1.7.1